packages/react/src/KeybindingHint/KeybindingHint.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {type ReactNode} from 'react'
```

Comment:
I was considering this but I didn't think it made sense to go through the draft phase for a component that's already stable and live in production. The API has been stable for about a year now without any breaking changes.

That said I have no strong feelings either way, but I would like to know what promotion criteria would need to be resolved before moving it to alpha so that I can follow up. Is it just a standard time period, or is there more to it?

Reply:
Alrighty, I'll move it